assets/app/scripts/services/navigate.js

```diff
@@ -48,6 +48,13 @@ angular.module("openshiftConsole")
         $location.path("project/" + encodeURIComponent(projectName) + "/create/next").search("name", name);
       },
 
+      toPodsForDeployment: function(deployment) {
+        $location.url("/project/" + deployment.metadata.namespace + "/browse/pods");
```

Comment:
Yeah. I copied the code as it was before without changing it. I'll test the compiled binary.

It would be good to understand how `$location` handles relative vs. absolute. It's not clear to me reading the Angular doc.

https://docs.angularjs.org/api/ng/service/$location#url